app/models/answer.rb

```diff
@@ -1,11 +1,11 @@
 class Answer < ApplicationRecord
   belongs_to :user
   belongs_to :question
-  has_many :comments, as: :commentable
+  has_many :comments, as: :commentable, dependent: :destroy
```

Comment:
This is great. Sure beats having to find all the associated objects and delete them manually.